tests/func/test_get.py

```diff
@@ -77,6 +110,51 @@ def test_get_a_dvc_file(erepo):
         Repo.get(erepo.root_dir, "some_file.dvc")
 
 
+# https://github.com/iterative/dvc/pull/2837#discussion_r352123053
+def test_get_full_dvc_path(erepo):
+    external_data_dir = erepo.mkdtemp()
+    external_data = os.path.join(external_data_dir, "ext_data")
+    with open(external_data, "w+") as fobj:
+        fobj.write("ext_data")
+
+    cur_dir = os.getcwd()
+    os.chdir(erepo.root_dir)
+    erepo.dvc.add(external_data)
+    erepo.dvc.scm.add(["ext_data.dvc"])
+    erepo.dvc.scm.commit("add external data")
+    os.chdir(cur_dir)
+
+    Repo.get(erepo.root_dir, external_data, "ext_data_imported")
+    assert os.path.isfile("ext_data_imported")
+    assert filecmp.cmp(external_data, "ext_data_imported", shallow=False)
```

Comment:
Should drop this actually, if out is cached then it doesn't matter whether it is external. This was inspired by complication in an implementation, once that is removed this will test non-existing code.

Reply:
@Suor Support for external outs is not going to be removed, we have active users already.

Reply:
I was talking about dropping a test, not external outs support. But you are probably right, there are many ways to break this even though outs are handled the same.